app/src/pages/inside/common/itemInfo/itemInfo.jsx

```diff
@@ -195,6 +197,12 @@ export class ItemInfo extends Component {
         </div>
 
         <div className={cx('additional-info')}>
+          {value.retentionPolicy === RETENTION_POLICY.IMPORTANT && (
+            <span className={cx('retention-policy')}>
+              <div className={cx('star-icon')}>{Parser(StarIcon)}</div>
```

Comment:
Let's follow the correct HTML structure and don't put `block` elements inside the `inline` elements.
I suppose the `div` here can be easily replaced by `span`.
[W3C recommendation](https://www.w3schools.com/html/html_blocks.asp#:~:text=Note%3A%20An%20inline%20element%20cannot%20contain%20a%20block%2Dlevel%20element!), [community discussions](https://stackoverflow.com/questions/1714121/block-level-elements-inside-inline-elements)